Dockerfile

```diff
@@ -61,3 +55,9 @@ RUN CHROME_MAJOR_VERSION=$(google-chrome --version | sed -E "s/.* ([0-9]+)(\.[0-
 #  && rm geckodriver-*.tar.gz
 
 ENTRYPOINT ["poetry" ,"run", "pytest"]
+
+FROM base as production 
+
+ENV FLASK_ENV=production
+
+CMD poetry run gunicorn "todo_app.app:create_app()" --bind 0.0.0.0:${PORT:-5000}
```

Comment:
Minor, but note that I think this has switched from a default port of 80 for Gunicorn to a default of 5000 - it looks like you don't have this bit documented in your Readme but this would be a good example of where we'd want to catch that we're changing the default behaviour!

Reply:
Will add to readme before merge